src/transformers/models/instructblip/modeling_instructblip.py

```diff
@@ -281,6 +281,8 @@ class InstructBlipPreTrainedModel(PreTrainedModel):
         r"language_model.decoder.embed_tokens.weight",
         r"language_model.lm_head.weight",
     ]
+    _keep_in_fp32_modules = ["wo"]
```

Comment:
Last comment: this is the same issue here (this comes from T5 if I'm not mistaken). This key should also be build dynamically.

Reply:
Makes sense, just updated it!